website/content/en/docs/advanced-topics/scorecard/kuttl-tests.md

```diff
@@ -4,160 +4,224 @@ weight: 50
+Initially your bundle may look like the following:
+
+```console
+$ tree bundle/
+bundle/
 ├── manifests
-│   ├── cache.example.com_memcacheds_crd.yaml
+│   |   ...
 │   └── memcached-operator.clusterserviceversion.yaml
 ├── metadata
 │   └── annotations.yaml
 └── tests
     └── scorecard
-        ├── config.yaml
-        └── kuttl
-            ├── kuttl-test.yaml
-            └── list-pods
-                ├── 00-assert.yaml
-                └── 00-pod.yaml
-            └── list-other
-                ├── 00-assert.yaml
-                └── 00-pod.yaml
+        └── config.yaml
 ```
 
- * `bundle/` - Contains bundle manifests and metadata under test.
- * `bundle/tests/scorecard/config.yaml` - Configuration yaml to define and run scorecard tests.
- * `tests/kuttl` - Contains tests written for kuttl to execute
- * `tests/kuttl/kuttl-test.yaml` - Contains the kuttl configuration, it is here that you would add any kuttl specific configuration settings that you might require.
- * `tests/kuttl/list-pods` - Contains a kuttl test case
- * `tests/kuttl/list-pods/00-assert.yaml` - Contains a kuttl test case assert
- * `tests/kuttl/list-pods/00-pod.yaml` - Contains a kuttl test case step
- * `tests/kuttl/list-other` - Contains another kuttl test case
-
-When the scorecard kuttl binary is executed, it will process all the test
-cases under the scorecard/kuttl directory within the bundle contents.
-
-## Configuring kuttl Tests in the Scorecard Configuration
+If you look at scorecard's [configuration file][scorecard-config] `bundle/tests/scorecard/config.yaml`,
+you'll notice nothing that refers to the scorecard-test-kuttl image described above. This is by design:
+kuttl tests should be enabled only once your operator is ready for in-cluster testing. Therefore the
+following patch specified in `config/scorecard/kustomization.yaml` was commented out at first:
 
-In the scorecard configuration file, you might have the following
-definition of what the selector `suite=kuttlsuite` will translate to:
 ```yaml
-stages:
-- tests:
-  - image: quay.io/operator-framework/scorecard-test-kuttl:dev
-    labels:
-      suite: kuttlsuite
-      test: kuttltest1
+patchesJson6902:
+...
+#- path: patches/kuttl.config.yaml
+#  target:
+#    group: scorecard.operatorframework.io
+#    version: v1alpha3
+#    kind: Configuration
+#    name: config
 ```
 
-This test configuration will execute the scorecard-test-kuttl
-image which executes kuttl.  The kuttl output is translated
-into scorecard compliant output which is displayed back to the
-end user along with any other test results.
+Now that you've written a robust test suite and your operator is ready for testing, uncomment the above
+patch and regenerate your bundle:
 
-With the above kuttl test configuration, you can execute that
-kuttl test using scorecard as follows:
-```bash
-operator-sdk scorecard <bundle_dir_or_image> --selector=suite=kuttlsuite 
+```sh
+make bundle IMG=<operator image>
```

Comment:
```suggestion
make bundle IMG=<some-registry>/<project-name>:<tag>
```

Reply:
I personally prefer `<operator image>` because the image tag is unimportant here (and is discussed in-depth in the OLM integration docs)

Reply:
The problem is that <operator image> do no make clear that the image requires to be in <some-registry> /pullable. I'd prefer we follow up a standard if the tag is not important then maybe `IMG=<some-registry>/<project-name>`. However, it is a nit I will not block this PR because of this.